snap-app/src/wordsearch/wordsearch.js

```diff
@@ -191,17 +187,19 @@ export default class Wordsearch extends React.Component {
     }
 
     hashStringToColor(str) {
-        var hash = 0;
+        let hash = 0;
         for (let i = 0; i < str.length; i++) {
             hash = str.charCodeAt(i) + ((hash << 5) - hash);
         }
-        let colour = '#';
+        let color = '#';
         for (let i = 0; i < 3; i++) {
-            const value = (hash >> (i * 8)) & 0xFF;
-            colour += ('00' + value.toString(16)).substr(-2);
+            let value = (hash >> (i * 8)) & 0xFF;
+            if (value < 100) {
```

Comment:
This is sort of a hack to ensure the hashed background color is "light" enough for black text.
Otherwise might be too difficult to see when color is pasted to spreadsheets without alpha channel.

Reply:
Oh good call - could also use HSL and hardcode a good lightness
```
return `hsl(${hash % 359}, 50%, 75%)`;
```